app/static/pages/home/home.html

```diff
@@ -42,322 +46,464 @@ <h1>
-             alt="Get involved image">
-      </div>
+      <md-button class="oppia-foundation-green-button
+                          oppia-foundation-margin-bottom-40"
+                 href="/#!/partnerships">
+        Learn More
+      </md-button>
     </md-card-content>
   </md-card>
-  <md-card class="oppia-foundation-vertical-padding
-                  oppia-foundation-mobile-and-tablet-horizontal-padding
-                  oppia-foundation-gray-background"
-           layout-gt-md="row"
-           layout="column"
-           layout-align="space-around"
-           layout-fill>
-    <md-card-content class="home-page-display-size
-                            oppia-foundation-no-padding"
-                     layout-gt-md="row"
-                     layout="column">
-      <div flex-gt-md="40"
+  <md-card class="oppia-foundation-white-background
+                  oppia-foundation-vertical-padding
+                  oppia-foundation-horizontal-padding"
+           show-xs
+           show-sm
+           hide-gt-sm
+           layout-fill
            layout="column"
-           layout-align="center">
-        <img class="home-page-partner-image"
-             src="/assets/images/home/home_partner.png"
-             aria-label="Partner with us image"
-             alt="Partner with us image">
-      </div>
-      <md-content class="oppia-foundation-black-text
-                         oppia-foundation-gray-background
-                         home-page-description-text-box"
-                  flex
-                  layout="column"
-                  layout-align="center">
-        <h3 class="md-display-1 home-page-content-title">
-          Partner with us
-        </h3>
-        <p>
-          We are looking for organizations that want to join us on our journey
-          to transform how students learn online.
-        </p>
+           layout-align="center center">
+    <md-card-content layout-fill
+                     layout="column"
+                     layout-align="center center">
+      <md-content class="oppia-foundation-white-background">
+        <md-card-title class="oppia-foundation-no-padding">
+          <md-card-title-text>
+            <h3 class="md-display-1
+                       home-page-content-title">
+              Volunteer
+            </h3>
+          </md-card-title-text>
+        </md-card-title>
         <p>
-          With your help, we aim to improve our platform by conducting randomly
-          controlled trials, fund a dedicated team of lesson creators, and
-          manage the costs of scaling the Oppia project to serve learners
-          across the world.
+          We have a broad community of volunteers who play critical
+          roles in everything from software development to graphic
+          design and even marketing and outreach. If you are
+          passionate about education and giving back, don't hesitate
+          to reach out. Become a part of the Oppian family today!
```

Comment:
Should Oppian be Oppia here?

Reply:
So this chunk of text was given to me by Diana. On closer reading, I also think it should be Oppia.